hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/SchemaTransaction.java

```diff
@@ -454,6 +476,12 @@ protected <T extends SchemaElement> List<T> getAllSchema(HugeType type) {
     protected void removeSchema(SchemaElement schema) {
         LOG.debug("SchemaTransaction remove {} by id '{}'",
                   schema.type(), schema.id());
+        // System schema just remove from SystemSchemaStore in memory
+        if (schema.longId() < 0L) {
+            throw new IllegalStateException("Deletion of system metadata " +
```

Comment:
the olap propertykey may can't be store into SystemSchemaStore?
https://github.com/apache/incubator-hugegraph/pull/1891/files#diff-3441066751fc582fd8c8588d9d7bdc97deeab0a789c29e478708084b2f127018R390